.github/workflows/build+test+publish.yml

```diff
@@ -89,3 +87,36 @@ jobs:
         tag: ${{ github.ref }}
         overwrite: true
         file_glob: true
+
+
+  deployReleaseDocs:
+    needs: deployReleaseBinaries
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+
+    # old .NET required by old fornax version
+    - name: Setup old .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 3.1.x
+
+    - name: Restore tools
+      run: dotnet tool restore
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Run Fornax
+      run: dotnet fake build -t Docs
+    - name: Deploy (if tag)
+      if: startsWith(github.ref, 'refs/tags/')
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        personal_token: ${{ secrets.GITHUB_TOKEN  }}
+        publish_dir: ./docs/_public
+        publish_branch: gh-pages
+        force_orphan: true
```

Comment:
@webwarrior-ws I dont know what have you done but this diff contains so many things that don't belong to this PR; bad rebase or bad conflict resolution?

Reply:
Rebased again. Now there are no conflicts.